public/js/main.js

```diff
@@ -0,0 +1,84 @@
+function initMap() {
+  const ironhackMAD = {
+    lat: 40.39279917456607,
+    lng: -3.698590505452073,
+  };
+
+  const map = new google.maps.Map(document.getElementById("map"), {
+    zoom: 13,
+    center: ironhackMAD,
+  });
+
+  getPlaces(map)
+    .then(places => {
+      // console.log(places.data.places)
+       placePlaces(map, places.data.places)
+    })
+    .catch(err =>console.log(err))
+  }
```

Comment:
- Si lo modificamos en la ruta como te puse en el otro comentario, nos ahorramos un nivel de profundidad y no haría falta acceder a la propiedad `places`, porque `places.data` ya nos devuelve un array con los objetos que contiene.

```suggestion
  getPlaces(map)
    .then(places => {
      placePlaces(map, places.data)
    })
    .catch(err => console.log(err))
}
```